cartridges/app_custom_page_designer/cartridge/client/default/scss/experience/components/commerceLayouts/carousel.scss

```diff
@@ -0,0 +1,127 @@
+@import "~base/experience/variables";
+@import "bootstrap/scss/variables";
+@import "bootstrap/scss/mixins/breakpoints";
+@import "~base/experience/carousel/extraSmallCarousel";
+@import "~base/experience/carousel/mediumCarousel";
+@import "~base/experience/carousel/smallCarousel";
+
+.row {
+    flex-wrap: nowrap;
+}
+
+.carousel-item {
```

Comment:
Please use BS utility classes

Reply:
These classes come from the main carousel.scss, the only thing that I've added is the .row { flex-wrap: nowrap; }